.gitignore

```diff
@@ -7,3 +7,54 @@ npm-debug.log*
 .DS_Store
 cypress-git-repo
 debug.log
+.metadata/.lock
```

Comment:
that's OK, to add (eclipse) `.metadata` to ignore-list, but pls add just the folder, not each file